device/bgpexplicitnulllabelpolicysubtlv.yaml

```diff
@@ -0,0 +1,17 @@
+components:
+  schemas:
+    Device.BgpExplicitNullLabelPolicySubTlv:
+      description: >-
+        Container for BGP explicit null label policy sub tlv settings.
+      type: object
+      properties:
+        enable:
+          description: >-
+            Enables the explicit null label policy sub tlv
+          type: boolean
+          default: false
+        explicit_null_label_policy:
+          description: >-
+            The value of the explicit null label policy 
+          type: integer
```

Comment:
The type can be string and add enum as per snapshot

![ENLP](https://user-images.githubusercontent.com/48308607/109392713-ad2b5500-7943-11eb-91e1-d29e5582a7bd.JPG)